Dockerfile

```diff
@@ -39,23 +35,22 @@ RUN apt-get -qq update && \
     wget http://packages.osrfoundation.org/gazebo.key -O - | apt-key add - && \
     add-apt-repository ppa:kisak/kisak-mesa && \
     curl -sSL http://get.gazebosim.org | sh && \
-    update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-10 800 --slave /usr/bin/g++ g++ /usr/bin/g++-10 --slave /usr/bin/gcov gcov /usr/bin/gcov-10 && \
     apt-get -qq update && \
     apt-get -qq upgrade -y --no-install-recommends \
-    libgz-cmake3-dev=3.2.* \
```

Comment:
in official docs they pin major version https://gazebosim.org/docs/harmonic/release-features#garden